examples/02-plot/backface_props.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _backface_props_example:
+
+Setting Backface Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default front and backface rendering uses the same properties. In certain
+situations it can be useful to set different properties for backfaces than
+for frontfaces.
+
+One straightforward example is when a closed (or close enough) surface has a
+different color on the inside. Note that the notion of "inside" and "outside"
+depend on the orientation of the surface normals:
+"""
+# sphinx_gallery_thumbnail_number = 1
+import numpy as np
+
+import pyvista as pv
+
+mesh = pv.ParametricEllipsoid(min_v=np.pi / 2 - 0.2, max_v=np.pi / 2 + 0.2)
+
+# create a shifted copy with flipped normals
+mesh_inside_out = mesh.translate((0, 0, 1), inplace=False)
+mesh_inside_out.compute_normals(flip_normals=True, inplace=True)
+meshes = mesh + mesh_inside_out
+
+backface_params = dict(color='orangered')
+meshes.plot(color='aquamarine', backface_params=backface_params, smooth_shading=True)
+
+
+###############################################################################
+# A more interesting use case is helping visualize the orientation of complex,
+# self-intersecting surfaces. For instance :func:`Catalan's minimal surface
+# <pyvista.ParametricCatalanMinimal>` has a complex shape, and coloring the
+# front and backfaces differently helps viewers comprehend the intricate
+# structure of the surface.
+
+catalan = pv.ParametricCatalanMinimal()
+backface_params = dict(color='forestgreen', specular=1.0, specular_power=50.0)
+catalan.plot(color='dodgerblue', backface_params=backface_params, smooth_shading=True)
+
+
+###############################################################################
+# In case of non-orientable surfaces adding specific backface properties can
```

Comment:
```suggestion
# In case of non-orientable surfaces, adding specific backface properties can
```

Reply:
Is it possible that you made the suggestion on a stale state, before the previous commit? The "the" isn't added here yet and github is being weird when I try to commit...

Reply:
Github is being weird I think, or it is a cache issue (my money is on this).  I'll try again.

Reply:
Perfect, thanks. Also: [you seem to be right](https://shouldiblamecaching.com/) ;)